engine/src/main/java/io/zeebe/engine/state/deployment/DbWorkflowState.java

```diff
@@ -132,21 +124,8 @@ private void persistWorkflow(final long workflowKey, final WorkflowRecord workfl
     workflowVersion.wrapLong(workflowRecord.getVersion());
 
     workflowByIdAndVersionColumnFamily.put(idAndVersionKey, persistedWorkflow);
-  }
-
-  @Deprecated
-  private void persistWorkflow(
-      final long workflowKey,
-      final WorkflowRecord workflowRecord,
-      final DeploymentResource resource) {
-    persistedWorkflow.wrap(resource, workflowRecord, workflowKey);
-    this.workflowKey.wrapLong(workflowKey);
-    workflowColumnFamily.put(this.workflowKey, persistedWorkflow);
 
-    workflowId.wrapBuffer(workflowRecord.getBpmnProcessIdBuffer());
-    workflowVersion.wrapLong(workflowRecord.getVersion());
-
-    workflowByIdAndVersionColumnFamily.put(idAndVersionKey, persistedWorkflow);
+    versionManager.setValue(workflowRecord.getBpmnProcessId(), workflowRecord.getVersion());
```

Comment:
It seems that we have two column families that stores the latest version of the workflow: `WORKFLOW_VERSION` (`versionManager`) and `WORKFLOW_CACHE_LATEST_KEY` (`latestWorkflowColumnFamily`) :thinking: 
Can we get rid of one of these column families?

Reply:
ok this is weird. why we have that?

Reply:
can we do this separate I feel uncomfortable with that :see_no_evil:

Reply:
:laughing: Sure. We can remove the column in a follow-up PR :heavy_check_mark: